docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       POSTGRES_USER: cxrdev
       POSTGRES_PASSWORD: cxrdev
     healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      test: ["CMD-SHELL", "pg_isready -d cxr_dev -U cxrdev"]
```

Comment:
Little trick to chill Docker out since it tries to connect to a non-existent database using the wrong user.